packages/replay-next/components/console/ConsoleInput.tsx

```diff
@@ -22,8 +22,9 @@ import useTerminalHistory from "./hooks/useTerminalHistory";
 import styles from "./ConsoleInput.module.css";
 
 export default function ConsoleInput() {
+  const { executionPoint } = useContext(TimelineContext);
   return (
-    <ErrorBoundary fallback={<ErrorFallback />}>
+    <ErrorBoundary key={executionPoint} fallback={<ErrorFallback />}>
```

Comment:
I see why you're doing this, and it makes sense. I think we should do it slightly differently though. Else any input that's in the console when you change execution point (even if just stepping through a function) will be erased, which seems bad.

Reply:
I think maybe a better UX (though more complicated code) would be for us to increment the key only if:
1. an error has already occurred
2. the execution point changes

Reply:
I've added a `resetKey` prop to `ErrorBoundary` that does exactly that. Please have a look if my implementation of that behavior is correct.